src/number_theory/number_field/embeddings.lean

```diff
@@ -123,3 +124,133 @@ end
 end bounded
 
 end number_field.embeddings
+
+section place
+
+variables {A : Type*} [normed_division_ring A] {K : Type*} [field K] (φ : K →+* A)
+
+/-- An embedding into a normed division ring defines a place of `K` -/
+def number_field.place : absolute_value K ℝ :=
+{ to_fun := norm ∘ φ,
+  map_mul' := by simp only [function.comp_app, map_mul, norm_mul, eq_self_iff_true, forall_const],
+  nonneg' := by simp only [norm_nonneg, forall_const],
+  eq_zero' := by simp only [norm_eq_zero, map_eq_zero, forall_const, iff_self],
+  add_le' := by simp only [function.comp_app, map_add, norm_add_le, forall_const], }
+
+end place
+
+namespace number_field.complex_embeddings
+
+open complex number_field
+
+open_locale complex_conjugate
+
+variables {K : Type*} [field K]
+
+/-- The conjugate of a complex embedding as a complex embedding. -/
+def conjugate (φ : K →+* ℂ) : K →+* ℂ := ring_hom.comp conj_ae.to_ring_equiv.to_ring_hom φ
+
+@[simp]
+lemma conjugate_coe_eq (φ : K →+* ℂ) (x : K) : (conjugate φ) x = conj (φ x) := rfl
+
+lemma conjugate_conjugate_eq (φ : K →+* ℂ) :
+  conjugate (conjugate φ) = φ :=
+by { ext1, simp only [conjugate_coe_eq, function.comp_app, star_ring_end_self_apply], }
+
+instance : has_involutive_star (K →+* ℂ) :=
```

Comment:
I am sorry to bother you again, but this should go in the file about star algebras, and should be generalized replacing `ℂ` by any star algebra. Also, all the results that are true in such generality should go there. Feel free to open another PR (you can also include `star_iff`). I promise a quick review :)

Reply:
No problem. You are definitely right. I do not have yet the reflex of trying to prove things to the highest level of generality, but that's what makes mathlib so great. Today is a bit of a busy day for me, but I'll get to it this evening or tomorrow.

Reply:
#17904

Reply:
The PR with the results about `star` is on its way to being merge (thanks to @eric-wieser), but now I have trouble using its content. I have merged it with this PR but I cannot get the instance to work in `embeddings.lean`. At the moment, I have a copy of the instance on line 151 of the file and everything compiles fine. But if I comment out the instance, then it fails saying in cannot synthesize type class instance for has_star (K →+* ℂ). Do I need to do something special to make the instance work?

Reply:
Have you recompiled everything? You can also wait for CI to build the oleans and download them (`leanproject get-cache`).

Reply:
I thought I did recompile all the dependencies of `embeddings.lean` but I'll double check. Thanks!

Reply:
There seems to be something wrong with the other PR #17904. It fails with the quite cryptic error (at least for me):
```
Error: /home/lean/actions-runner/_work/mathlib/mathlib/src/algebra/module/linear_map.lean:11:0: invalid import: algebra.star.basic
invalid object declaration, environment already has an object named 'assert_not_exists._checked._fresh.10.7'
```

Reply:
I made an error in #17904 (which is now merged). I defined `{S : Type u}` in `ring_hom.has_involutive_star` (and other functions) which forces `R` and `S` to be in the same universe and explain why it didn't work when it tried to use it in this PR. I fixed the error in this PR by changing to `{S : Type*}`. Sorry about that.  I should be more cautious when dealing with universes.